Resources/Prototypes/Entities/Mobs/Customization/Markings/scars.yml

```diff
@@ -2,7 +2,7 @@
   id: ScarEyeRight
   bodyPart: Head
   markingCategory: Head
-  speciesRestriction: [Human, Dwarf, Felinid, Harpy, Oni, Rodentia, Feroxi]  # Delta V - Felinid, Oni, Harpy, Rodentia, Feroxi
+  speciesRestriction: [Human, Dwarf, Felinid, Harpy, Oni, Rodentia, Feroxi, Vulpkanin]  # DeltaV - Felinid-Oni-Harpy-Rodentia-Feroxi-Vulpkanin
```

Comment:
just have it be Felinid-Vulpkanin for all of these theres no point to keeping this meme alive